src/schemas/transaction.yaml

```
@@ -1,3 +1,92 @@
+Transaction7702Signed:
+  type: object
+  title: EIP-7702 transaction.
+  required:
+    - type
+    - nonce
+    - to
+    - gas
+    - value
+    - input
+    - maxPriorityFeePerGas
+    - maxFeePerGas
+    - maxFeePerBlobGas
+    - accessList
+    - blobVersionedHashes
+    - chainId
+    - authorizationlist
+  properties:
+    type:
+      title: type
+      type: string
+      pattern: ^0x2$
+    nonce:
+      title: nonce
+      $ref: '#/components/schemas/uint'
+    to:
+      title: to address
+      oneOf:
+        - title: Contract Creation (null)
+          type: 'null'
+        - title: Address
+          $ref: '#/components/schemas/address'
+    gas:
+      title: gas limit
+      $ref: '#/components/schemas/uint'
+    value:
+      title: value
+      $ref: '#/components/schemas/uint'
+    input:
+      title: input data
+      $ref: '#/components/schemas/bytes'
+    maxPriorityFeePerGas:
+      title: max priority fee per gas
+      description: Maximum fee per gas the sender is willing to pay to miners in wei
+      $ref: '#/components/schemas/uint'
+    maxFeePerGas:
+      title: max fee per gas
+      description: The maximum total fee per gas the sender is willing to pay (includes the network / base fee and miner / priority fee) in wei
+      $ref: '#/components/schemas/uint'
+    maxFeePerBlobGas:
+      title: max fee per blob gas
+      description: The maximum total fee per gas the sender is willing to pay for blob gas in wei
+      $ref: '#/components/schemas/uint'
+    accessList:
+      title: accessList
+      description: EIP-2930 access lists
+      $ref: '#/components/schemas/AccessList'
+    chainId:
+      title: chainId
+      description: Chain ID that this transaction is valid on.
+      $ref: '#/components/schemas/uint'
+    authorizationList:
```

```suggestion
    authList:
```

I think this is equally clear and much shorter.